test/test_jit_string.py

```diff
@@ -1,5 +1,6 @@
 from test_jit import JitTestCase
```

Comment:
these tests are not run in oss CI. You need to whitelist them in `run_tests.py`.